From b65d906b9047b2e357c5a82f09715382100494a1 Mon Sep 17 00:00:00 2001
From: Evgeny Karpov <evgeny.karpov@microsoft.com>
Date: Mon, 4 Nov 2024 14:47:51 +0100
Subject: [PATCH 01/38] aarch64: Bypass some warnings in MinGW

The patch bypasses missing prototypes warnings in MinGW until it is
fixed.

libgcc/ChangeLog:

	* config/aarch64/t-mingw: Update.
---
 libgcc/config/aarch64/t-mingw | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/libgcc/config/aarch64/t-mingw b/libgcc/config/aarch64/t-mingw
index a72a2f701774..27d28d42c914 100644
--- a/libgcc/config/aarch64/t-mingw
+++ b/libgcc/config/aarch64/t-mingw
@@ -1,2 +1,2 @@
-# Bypass hidden attribute warnings in MinGW until it is implemented
-LIBGCC2_CFLAGS += -Wno-error=attributes
+# Bypassing hidden attribute warnings in MinGW until it is implemented
+LIBGCC2_CFLAGS += -Wno-error=attributes -Wno-error=missing-prototypes
-- 
2.51.0

